tools_considered_and_the_screening_decisions.md

```diff
@@ -0,0 +1,178 @@
+|SRDB.PRO                                                                       |         1|         1|         0|no RITL-based AL cycle                                                      |
+|SRDR+                                                                          |         1|         0|         0|no RITL-based AL cycle                                                      |
+|StArt                                                                          |         1|         1|         0|no RITL-based AL cycle                                                      |
+|SWIFT-Review                                                                   |         1|         1|         0|no RITL-based AL cycle                                                      |
+|Syras                                                                          |         0|         1|         0|no RITL-based AL cycle                                                      |
+|SyRF: Systematic Review Facility                                               |         1|         1|         0|no RITL-based AL cycle                                                      |
+|SysRev                                                                         |         1|         1|         0|no RITL-based AL cycle                                                      |
+|Systematic Review Accelerator / SR-Accelerator / SRA                           |         1|         1|         0|no RITL-based AL cycle                                                      |
+|TaskExchange                                                                   |         1|         1|         0|no RITL-based AL cycle                                                      |
+|TextBlob: Simplified Text Processing                                           |         0|         1|         0|no software                                                                 |
+|Thoth                                                                          |         1|         1|         0|no RITL-based AL cycle                                                      |
+|Trialstreamer                                                                  |         1|         0|         0|limited to COVID19                                                          |
+                           
+
+
+
+## Search 1
+
+We consider screening tools that were part of a comparison in papers that cite the 2020 paper by Harrison et al. and/or the 2021 paper by Van de Schoot et al. as of December 2022.
+
+On 6/12/2022, the Scopus database contained 40 papers that cited the Harrison 2020 paper. Of these results, 22 could be downloaded directly from the Scopus website. Of the remaining 18, 17 PDFs could be retrieved from different websites and of only 1 article no PDF version was available (Bravo 2021), so the htm-link was saved instead. 
```

Comment:
I'm not sure if this is of any additional value, but is it fine if we just search in Scopus?

Google Scholar finds **91 articles** that cite Harrison
![image](https://user-images.githubusercontent.com/65157657/209163372-c4983169-5efb-4609-8922-40e0f4da8312.png)


and **117 articles** that cite van de Schoot:
![image](https://user-images.githubusercontent.com/65157657/209163573-6e4c50ab-aaa4-4491-a6c8-1c14a28917a3.png)

Reply:
Yes, because scholar google is not a persistent database